packages/babel-core/src/util.js

```diff
@@ -22,140 +15,3 @@ export function message(opts: Object, msg: string) {
 
   return `[BABEL] ${opts.filename || "unknown"}: ${msg}`;
 }
-
-/**
- * Test if a filename ends with a compilable extension.
- */
-
-export function canCompile(filename: string, altExts?: Array<string>): boolean {
```

Comment:
I ended up backing down on removing the extensions, just canCompile. Expose `babel.DEFAULT_EXTENSIONS` so we can still drop `util`.